CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Unreleased
+- Due to changes in the implementation of `typing_extensions.Protocol`,
+  `typing.runtime_checkable` can now be used on `typing_extensions.Protocol`
```

Comment:
Can you add a versionchanged to the documentation for this?

Reply:
I had a go in 00f568b, but I'm not really sure we should be advertising this too widely. Users shouldn't really be depending on being able to use `typing.runtime_checkable` on `typing_extensions.Protocol`; it's more of a "nice-to-have" (and it will simplify the implementation for _us_ when we drop support for Python 3.7).

Reply:
Yes, that documentation is very confusing to read, we're probably better off without it. Sorry for the churn!

Reply:
No worries!